cellumata_syntax-v2/cellmata.g4

```diff
@@ -9,14 +9,24 @@ const_ident : IDENT ;
 // World
 world_dcl : STMT_WORLD BLOCK_START world_size world_tickrate? world_cellsize? BLOCK_END ;
 world_size : 'size' ASSIGN world_size_dim (LIST_SEP world_size_dim)?;
-world_size_dim : DIGITS SQ_BRACKET_START ('wrap' | 'edge' ASSIGN IDENT) SQ_BRACKET_END ;
-world_tickrate : 'tickrate' ASSIGN DIGITS ;
-world_cellsize : 'cellsize' ASSIGN DIGITS ;
+world_size_dim : DIGITS SQ_BRACKET_START world_size_dim_finite SQ_BRACKET_END # dimFinite ;
+world_size_dim_finite
+    : 'wrap' # dimFiniteWrapping
+    | 'edge' ASSIGN IDENT # dimFiniteEdge
+    ;
+world_tickrate
+            : 'tickrate' ASSIGN world_tickrate_value # tickrate
```

Comment:
Inconsistent case-scheme. `tickrate` here and `cellSize` below. Dromedary case should be consistent.

Reply:
Both lowercase would be fine, too.